### PR TITLE
20.11 fb necropsy request form

### PR DESCRIPTION
**Pull request by Ohsudev**
#### Changes
Modified Necropsy request form to include additional new input columns, and input control selections.

### CODE REVIEW

File: onprc_ehr/resources/web/onprc_ehr/panel/TissueRequestInstructionsPanel.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2014 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+Ext4.define('ONPRC_EHR.panel.NecropsuRequestInstructionsPanel', {
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/web/onprc_ehr/panel/TissueRequestInstructionsPanel.js]: Typo here - likely won't cause immediate problems, but good to correct anyway

File: onprc_ehr/resources/web/onprc_ehr/model/sources/NecropsyRequest.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            },
+            chargetype: {
+                defaultValue: 'DCM: Pathology Services',
+                hidden: true
+            },
+            date: {
+                xtype: 'xdatetime',
+                header: 'Start Date',
+                extFormat: 'Y-m-d H:i',
+                defaultValue: (new Date()).format('Y-m-d 8:0')
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/web/onprc_ehr/model/sources/NecropsyRequest.js]: This is the same unexpected date format that I asked Lakshmi about. Is the goal to default to 8:00AM?

**Review by labkey-jeckels — APPROVED** [review]
Two items that you may want to update but don't block merging this either.

**Comment by Ohsudev** [comment]
I will make the necessary corrections.  Thank you.

Raymond

From: Josh Eckels ***@***.***>
Sent: Tuesday, March 23, 2021 11:13 AM
To: LabKey/onprcEHRModules ***@***.***>
Cc: Raymond Blasa ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [LabKey/onprcEHRModules] 20.11 fb necropsy request form (#126)


@labkey-jeckels approved this pull request.

Two items that you may want to update but don't block merging this either.

________________________________

In onprc_ehr/resources/web/onprc_ehr/panel/TissueRequestInstructionsPanel.js<https://github.com/LabKey/onprcEHRModules/pull/126#discussion_r599818897>:

> @@ -0,0 +1,27 @@

+/*

+ * Copyright (c) 2014 LabKey Corporation

+ *

+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0

+ */

+Ext4.define('ONPRC_EHR.panel.NecropsuRequestInstructionsPanel', {

Typo here - likely won't cause immediate problems, but good to correct anyway

________________________________

In onprc_ehr/resources/web/onprc_ehr/model/sources/NecropsyRequest.js<https://github.com/LabKey/onprcEHRModules/pull/126#discussion_r599819109>:

> +    },

+    byQuery: {

+        'study.encounters': {

+            type: {

+                defaultValue: 'Necropsy',

+                hidden: true

+            },

+            chargetype: {

+                defaultValue: 'DCM: Pathology Services',

+                hidden: true

+            },

+            date: {

+                xtype: 'xdatetime',

+                header: 'Start Date',

+                extFormat: 'Y-m-d H:i',

+                defaultValue: (new Date()).format('Y-m-d 8:0')

This is the same unexpected date format that I asked Lakshmi about. Is the goal to default to 8:00AM?

**Pull request closed without merging**